fetch.bs

```diff
@@ -4329,6 +4329,31 @@ steps:
 <var>response</var>, a <a for=/>global object</a> <var>global</var>, and a <a for=/>string</a>
 <var>initiatorType</var> (default "<code>other</code>"), run these steps:
 
+<ol>
+ <li><p>Let <var>taoPass</var> be true if <var>response</var>'s
+ <a for=response>timing allow passed flag</a> is set; otherwise false.
+
+ <li>Set <var>response</var>'s <a for="response">timing info</a> to the
+ result of calling <a>report timing</a> given <var>response</var>, <var>global</var>,
+ <var>initiatorType</var>, and <var>taoPass</var>.
+</ol>
+
+<p>To <dfn export>report timing with origin</dfn> given a <a for=/>response</a>
+<var>response</var>, a <a for=/>global object</a> <var>global</var>, an <a for=/>origin</a>
+<var>origin</var>, and a <a for=/>string</a> <var>initiatorType</var> (default
+"<code>other</code>"), <a>report timing</a> for <var>response</var>,
+<var>global</var>, <var>initiatorType</var>, and the result of calling <a>TAO origin check</a> with
+<var>response</var> and the result of <a lt="ASCII serialization of an origin">serializing</a>
+<var>origin</var>.
+
+<p class=note>This algorithm is used by <cite>HTML</cite> to report resource timing for
+iframes, as iframes report both to the iframe's loaded document as navigation timing and to the
+containing document as resource timing.
```

Comment:
This needs [[HTML]] or some such at the end. It should also use more formal language to describe these concepts. Perhaps "report resource timing documents nested through `iframe` elements"?

Reply:
Fixed

Reply:
It seems the language comment wasn't addressed/pushed?

Reply:
Oops, oversight. Sorry for the noise.